src/js/components/DataTable/DataTable.js

```diff
@@ -219,62 +223,35 @@ const DataTable = ({
     console.warn('DataTable cannot combine "size" and "resizeble".');
   }
 
+  const [page, showItem] = normalizeShow(showProp, 'dataTable');
```

Comment:
Something feels a bit odd, to have to pass "dataTable" to this function, like every time we call this function from a new component, we have to make the function aware of that component. Can we come up with something that makes more sense from the context of `normalizeShow`?

Reply:
I think we can probably simplify this function now. Previously we were having a different behavior to normalize "show" for Pagination, so context was helping with clarity, but I think we can omit. I'll update.

Reply:
Done.